include/uapi/linux/vhost.h

```diff
@@ -93,6 +93,8 @@
 #define VHOST_SET_BACKEND_FEATURES _IOW(VHOST_VIRTIO, 0x25, __u64)
 #define VHOST_GET_BACKEND_FEATURES _IOR(VHOST_VIRTIO, 0x26, __u64)
 
+#define VHOST_SET_RUNNING _IOW(VHOST_VIRTIO, 0x61, int)
```

Comment:
@lyakh typo in the commit "inform it of a status changem"

Reply:
@ranj063 I keep hitting "m" instead of a "," - seems like a finger calibration error, thanks